### ISSUE TITLE
Watchlist Value-WindowsEventsDictionary failing to import

### ISSUE THREAD

**Issue by jrlane** [ep3p/Sentinel_KQL#2]
Any ideas why this watchlist fails to import as a watchlist? The sentinel message is not super helpful.

"Watchlist [Value-WindowsEventsDictionary] could not be properly saved. "

At first, I thought it might have been because of some non ascii chars (the " and '), but after removing those it still fails to import just the same.

**Comment by ep3p** [comment]
Sorry @jrlane I made a mistake. Try now.

In a watchlist the Search column cannot be empty, there is 1 line that was empty. It is the line of "ms-Mcs-AdmPwd".

In this page (https://www.powershellgallery.com/packages/SDDLParser/0.5.0/Content/SDDLParserADObjects.ps1) "ms-Mcs-AdmPwd" has the value "4c9928d7-d725-4fa6-a109-aba3ad8790e5", but I have never found this value in Windows Security Events, instead I found "ms-Mcs-AdmPwd" has another value.

I BELIEVE "ms-Mcs-AdmPwd" can have a different value in each AD tenant. So I did not upload the specific value I found in Windows Security Events, just in case someone could consider it confidential.

I am going to put the value "4c9928d7-d725-4fa6-a109-aba3ad8790e5" (that is already public) in the Watchlist for "ms-Mcs-AdmPwd", but maybe it will not be the correct value for your Windows Security Events.
